src/main/java/com/nerodesk/om/Account.java

```diff
@@ -41,6 +42,24 @@
 @Immutable
 public interface Account {
     /**
+     * Dummy implementation.
+     */
+    Account DUMMY = new Account() {
```

Comment:
Would it be better to create a class `DummyAccount` instead of constant? In this case we'll use different objects instead of the same which sounds better in terms of OOP, doesn't it?